src/systemcmds/param/param.c

```diff
@@ -331,6 +342,20 @@ do_show(const char *search_string)
 }
 
 static int
+do_find(const char *name)
+{
+	param_t ret = param_find_no_notification(name);
+
+	if (ret == PARAM_INVALID) {
+		warnx("Parameter %s not found", name);
+		return 1;
+	}
+
+	printf("Found param %s at index %lu\n", name, ret);
```

Comment:
To get rid of the CI error, you can use ...%" PRIxPTR "\n", ... (from <inttypes.h>)